js(analytics): update pinpoint service provider configuration options for V6 (#6261)
aws-amplify/docs: src/fragments/lib/analytics/js/existing-resources.mdx

Amplify.configure({
  ...Amplify.getConfig(),
  Analytics: {
    ...Amplify.getConfig().Analytics,
    Pinpoint: {
      // REQUIRED -  Amazon Pinpoint App Client ID
      appId: 'XXXXXXXXXXabcdefghij1234567890ab',

      // REQUIRED -  Amazon service region
      region: 'us-east-1',

>       // OPTIONAL - The buffer size for events in number of items.

Reviewer: ```suggestion
      // OPTIONAL - How many events can be buffered at once.
```

Author: The other Analytics service providers are using the same description for these options. I can update all of them at once in another PR. 

ref: https://github.com/aws-amplify/docs/blob/next-release/main/src/fragments/lib/analytics/js/streaming.mdx#L35-L45

      bufferSize: 1000,

>       // OPTIONAL - The number of events to be deleted from the buffer when flushed.

Reviewer: ```suggestion
      // OPTIONAL - How many events will be flushed from the buffer per batch.
```
